src/ripple/protocol/XRPAmount.h

```diff
@@ -63,6 +68,14 @@ class XRPAmount
     {
     }
 
+    template <class Integer,
+        class = typename std::enable_if_t <
+            std::is_integral<Integer>::value>>
+    XRPAmount (tagged_integer<Integer, DropsTag> drops)
+        : drops_ (static_cast<std::int64_t> (drops.value()))
```

Comment:
nit: isn't drops.value() already an int64_t?

Reply:
No, it's a `uint64_t`.